src/screens/Voucher/VoucherIndexScreen.tsx

```diff
@@ -83,9 +90,30 @@ export const VoucherIndexScreen = ({ navigation, route }: StackScreenProps<any>)
                 </Wrapper>
               )}
 
-              <Wrapper>
-                <RegularText>Add login component here</RegularText>
-              </Wrapper>
+              {!isLoggedIn && (
+                <Wrapper>
+                  <WrapperVertical>
+                    <BoldText>{texts.voucher.indexLoginTitle}</BoldText>
+                  </WrapperVertical>
+
+                  <WrapperVertical style={styles.noPaddingTop}>
+                    <RegularText>{texts.voucher.indexLoginDescription}</RegularText>
+                  </WrapperVertical>
+
+                  <Button
+                    title={texts.voucher.loginButton}
+                    onPress={() => navigation.navigate(ScreenName.VoucherLogin)}
+                  />
+                </Wrapper>
+              )}
+
+              {count > 0 && (
+                <Wrapper style={styles.noPaddingTop}>
+                  <BoldText>
+                    {count} {texts.voucher.results}
```

Comment:
this does not work for singular/plural depending on 1 oder more -> 1 Ergebnis / 3 Ergebnisse

do we need the number here? if we come from the categories screen, we have the number there.
if we look at the latest vouchers, does it help users if there are 5 or 7?

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/b6681ea1e42b1c3ca222c3fc4b223224df386078